lib/web/ui/server.go

```diff
@@ -368,3 +371,27 @@ func MakeDesktopServices(windowsDesktopServices []types.WindowsDesktopService) [
 
 	return desktopServices
 }
+
+// stripProtocolAndPort returns only the hostname of the uri.
+// Handles uri's with no protocol eg: for some database connection
+// endpoint the uri can be in the format "hostname:port".
+func stripProtocolAndPort(uri string) string {
```

Comment:
I realize this is a backport and is already approved, but out of curiousity why are we not using `url.Parse` here?

Reply:
:sweat_smile: https://github.com/gravitational/teleport/pull/19063#discussion_r1041211774

Reply:
Hah! Thanks Marco! 😄

Reply:
sorry for my subpar grammar :sweat_smile:, fixed the grammer here and sneaked it in here for master: https://github.com/gravitational/teleport/pull/19285/commits/a9be35ab61fb56e1ef0e6aa3976280958e5555b9